signingscript/docker.d/passwords.yml

```diff
@@ -247,6 +254,18 @@ in:
              {"$eval": "AUTOGRAPH_FENNEC_RELEASE_PASSWORD"},
              ["autograph_apk"]
             ]
+        project:mobile:fennec-profile-manager:releng:signing:cert:fennec-nightly-signing:
```

Comment:
Both of these are present already in this type of workers, so no SOPS or cloudops-infra changes needed either.